csharp/src/Apache.Arrow/ArrowBuffer.BitmapBuilder.cs

```diff
@@ -138,6 +138,24 @@ public BitmapBuilder AppendRange(IEnumerable<bool> values)
                 return this;
             }
 
+            /// <summary>
+            /// Append multiple bits.
+            /// </summary>
+            /// <param name="value">Value of bits to append.</param>
+            /// <param name="length">Number of times the value should be added.</param>
+            /// <returns>Returns the builder (for fluent-style composition).</returns>
+            public BitmapBuilder AppendRange(bool value, int length)
```

Comment:
Should check that `length` is non-negative.

Reply:
Fixed